lib/builder.js

```diff
@@ -122,7 +123,7 @@ function writeOutputFile(opts, output, basePath) {
   });
 }
 
-exports.inlineSourceMap = function(output) {
+var inlineSourceMaps = exports.inlineSourceMap = function(output) {
```

Comment:
Better tests will catch the kinds of slips I made. Missed that typo. Feeling a squash coming on :)